client/components/global/Header.jsx

```diff
@@ -36,9 +36,19 @@ const Header = ({ user, title, variant, background }) => {
               Create a Book Club
             </Nav.Link>
           </NavDropdown>
-          <Nav.Link as={Link} to='/subscriptions' variant={variant}>
-            Subscriptions
-          </Nav.Link>
+          {user ? (
+            user.suscriptionTier ? (
+              <></>
+            ) : (
+              <Nav.Link as={Link} to='/subscriptions' variant={variant}>
+                Subscriptions
+              </Nav.Link>
+            )
+          ) : (
+            <Nav.Link as={Link} to='/subscriptions' variant={variant}>
+              Subscriptions
+            </Nav.Link>
+          )}
```

Comment:
It looks like this can be significantly simplified. something like user && user.subscriptionTier ? </> : <Nav.Link.... etc.

Reply:
Tried didnt work for some reason